space_age/app/views/packages/index.html.erb

```diff
@@ -1,6 +1,29 @@
-<h1>Packages Index</h1>
+<div class="orbit" role="region" aria-label="Favorite Space Pictures" data-orbit>
+  <ul class="orbit-container">
+    <button class="orbit-previous" aria-label="previous"><span class="show-for-sr">Previous Slide</span>&#9664;</button>
+    <button class="orbit-next" aria-label="next"><span class="show-for-sr">Next Slide</span>&#9654;</button>
+    <li class="orbit-slide is-active">
+      <%= image_tag("slide_1.jpeg") %>
+    </li>
+    <li class="orbit-slide">
+      <%= image_tag("slide_1.jpeg") %>
+    </li>
+    <li class="orbit-slide">
+      <%= image_tag("slide_2.jpeg") %>
+    </li>
+    <li class="orbit-slide">
+      <%= image_tag("slide_3.jpeg") %>
+    </li>
+  </ul>
+</div>
 
 <% @packages.each do |package| %>
-  <%= link_to package.title, package %>
-  <%= link_to(image_tag(package.image, alt: package.image), package_path(package))%>
+<%= link_to package.title, package %>
+<%= link_to(image_tag(package.image, alt: package.image), package_path(package))%>
 <%end %>
+
+<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+<script src="http://dhbhdrzi4tiry.cloudfront.net/cdn/sites/foundation.js"></script>
+<script>
+      $(document).foundation();
```

Comment:
We shouldn't need those script tags as that is all being called in the application. Also the second one is the cdn, we have all of that locally

We can take a look tomorrow!